<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="ent1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>661</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="left1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>◄</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>►</string>
    </property>
   </widget>
   <widget class="QListWidget" name="favs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>661</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 8pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
